akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -253,10 +255,9 @@ object Flow {
    *
    * '''Cancels when''' downstream cancels
    */
-  @Deprecated
   @deprecated(
-    "Use lazyInitAsync instead. (lazyInitAsync returns a flow with a more useful materialized value.)",
-    "2.5.12")
+    "Use 'Flow.completionStageFlow' in combination with prefixAndTail(1) instead, see `completionStageFlow` operator docs for details",
```

Comment:
It seems a bit weird that we now have created helper methods for many of the cases that would be relatively simple to combine but remove this one which only has a clunky replacement using `prefixAndTail`. On the other hand, the more complex the replacement, the more likely it is that people would like to customize the behavior in the first place. It's hard to say without having some usage numbers.

Reply:
Just to be sure, this particular method had a weird signature anyway, so I'm glad to see it go...